heltour/tournament/create_teams.py

```diff
@@ -0,0 +1,386 @@
+import random
```

Comment:
I don't love this file name; can we call it teamgen.py instead (similar to pairinggen.py)?